parley_data_gen/Cargo.toml

```diff
@@ -13,7 +13,7 @@ icu_locale = { workspace = true }
 icu_properties = { workspace = true, features = ["compiled_data"] }
 icu_collections = { workspace = true, features = ["databake"] }
 icu_provider_source = { workspace = true, features = ["networking", "use_wasm"] }
-unicode_data = { workspace = true, default-features = false, features = ["datagen"] }
+parley_data = { workspace = true, default-features = false, features = ["datagen"] }
```

Comment:
Can you please fix the ordering? Unfortunately these dependencies are not alphabetical or otherwise ordered, but grouping the `icu_*` deps together helps improve the ability for one to scan the deps